extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
```

Comment:
Right I agree that we should limit this to adsense and doubleclick.  I believe the selector would have to be:

amp-ad[data-a4a-upgrade-type='amp-ad-network-adsense-impl] > iframe, amp-ad[data-a4a-upgrade-type='amp-ad-network-doubleclick-impl] > iframe